README.md

```diff
@@ -74,7 +74,36 @@ To integrate the CAP Operator Plugin into your project, follow these steps:
 
 4. After filling all the design-time information in `values.yaml`, run `cds build`. The final chart will be generated in the `gen` folder within your project directory.
 
-5. Now to deploy the application, you can pass the runtime values in a separate `runtime-values.yaml` file and deploy the chart using the following command:
+5. To deploy the application, you need to create `runtime-values.yaml` with all the runtimes values as mentioned above. For that you can make use of the plugin itself. The plugins provides two ways to generate the runtime values file -
+
+    * **Interactive Mode** - This mode will ask you for all the runtime values one by one. To use this mode, run the following command:
+
+        ```sh
+        npx cap-op-plugin generate-runtime-values --via-prompts
```

Comment:
npx cap-op-plugin generate-runtime-values
(or optionally --with-prompts as argument)

This mode is defaulted when no arguments are passed

Reply:
Removed --with-prompts and made it default